dali/operators/debug/dump_image.cc

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2017-2018, NVIDIA CORPORATION. All rights reserved.
+// Copyright (c) 2017-2021, NVIDIA CORPORATION. All rights reserved.
```

Comment:
```suggestion
// Copyright (c) 2017-2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Reply:
I'm going to keep it as is till we decide against doing it in one go.